apps/transport/lib/transport_web/templates/backoffice/page/index.html.eex

```diff
@@ -21,6 +21,7 @@
     <th></th>
     <th></th>
     <th></th>
+    <th></th>
```

Comment:
is this meant to be here?

Reply:
yes without it the backoffice is:
![image](https://user-images.githubusercontent.com/3987698/61300541-b2b94300-a7d1-11e9-9364-6c140ce50b84.png)


and with it:
![image](https://user-images.githubusercontent.com/3987698/61300619-d41a2f00-a7d1-11e9-9f2c-528b313de7ef.png)